content/en/docs/reference/labels-annotations-taints/_index.md

```diff
@@ -580,6 +580,14 @@ receiving traffic for the Service from the moment the kubelet starts all contain
 and marks it _Running_, til the kubelet stops all containers and deletes the pod from
 the API server.
 
+### autoscaling.alpha.kubernetes.io/behavior (deprecated)
+
+Type: Annotation
+
+Used on: HorizontalPodAutoscaler
+
+This annotation was used to configure the scaling behavior of the Horizontal Pod Autoscaler (HPA) in earlier Kubernetes versions. It allowed to specify how the HPA should scale pods up or down, including setting stabilization windows and scaling policies.
```

Comment:
```suggestion
This annotation was used to configure the scaling behavior for a HorizontalPodAutoscaler (HPA) in earlier Kubernetes versions.
It allowed you to specify how the HPA should scale pods up or down, including setting stabilization windows and scaling policies.
```

Reply:
Sure, I can push those changes in right away